cmd/kops/integration_test.go

```diff
@@ -73,7 +73,7 @@ func TestComplex(t *testing.T) {
 func TestMinimalCloudformation(t *testing.T) {
 	//runTestCloudformation(t, "minimal.example.com", "../../tests/integration/minimal", "v1alpha0", false)
 	//runTestCloudformation(t, "minimal.example.com", "../../tests/integration/minimal", "v1alpha1", false)
-	runTestCloudformation(t, "minimal.example.com", "../../tests/integration/minimal", "v1alpha2", false)
+	runTestCloudformation(t, "minimal.example.com", "../../tests/integration/cloudformation", "v1alpha2", false)
```

Comment:
minimal is the only one where we actually test CF output - in theory we should test them all, but I'm not convinced it is worth it.